src/api/app/controllers/webui/package_controller.rb

```diff
@@ -865,10 +866,11 @@ def set_initial_offset
   def update_build_log
     check_ajax
 
-    @project = params[:project]
-    @package = params[:package]
-    @arch = params[:arch]
-    @repo = params[:repository]
+    # Make sure objects don't contain invalid chars (eg. '../')
+    @project = Project.find_by(name: params[:project]).try(:name)
+    @package = Package.find_by(name: params[:package]).try(:name)
```

Comment:
I don't get that line. It is searching for a package with that name in *any* project?